src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/Microsoft.NET.Sdk.Functions.targets

```diff
@@ -21,6 +21,12 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <WarnOnPackingNonPackableProject Condition="'$(WarnOnPackingNonPackableProject)' == '' and '$(IsPackable)' == 'false'">true</WarnOnPackingNonPackableProject>
     <IsZipDeploySupported>true</IsZipDeploySupported>
     <MSBuildFunctionsTargetsPath>$(MSBuildExtensionsPath)\Microsoft\VisualStudio\Managed.Functions\</MSBuildFunctionsTargetsPath>
+
+    <_ToolingSuffix></_ToolingSuffix>
+    <_ToolingSuffix Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(TargetFrameworkVersion)' == 'v8.0'">net8-in-process</_ToolingSuffix>
```

Comment:
where do `TargetFrameworkIdentifier` and `TargetFrameworkVersion` coming from? are those set in an upstream targets file?

Reply:
I believe these are standard MSBuild things, but I'm having trouble finding reference for them. I picked this all up from the worker: https://github.com/Azure/azure-functions-dotnet-worker/blob/455e59d097cdf0e45cd2a981ec1f2cfee4902382/sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets#L16